google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -142,6 +142,12 @@ def bq_to_arrow_struct_data_type(field):
     return pyarrow.struct(arrow_fields)
 
 
+def bq_to_arrow_range_data_type(field):
+    element_type = field.element_type.upper()
+    arrow_element_type = _pyarrow_helpers.bq_to_arrow_scalars(element_type)()
```

Comment:
do we need to do validation here?  None-check?

Reply:
Great point, I will add a None-check here

Reply:
I added it, as well as the unit tests in `test__pandas_helpers.py`.